pwiz_tools/Skyline/Controls/Graphs/GraphSummary.cs

```diff
@@ -269,7 +269,12 @@ public int CountCurves(Func<CurveItem, bool> isCounted)
         internal IEnumerable<SummaryGraphPane> GraphPanes
         {
             get { return graphControl.MasterPane.PaneList.OfType<SummaryGraphPane>(); }
-            set { graphControl.MasterPane.PaneList.Clear(); graphControl.MasterPane.PaneList.AddRange(value); }
+            set
+            {
+                graphControl.MasterPane.PaneList.OfType<SummaryGraphPane>().ForEach(panel => panel.OnClose(EventArgs.Empty));

```

Comment:
It is possible that a pane doesn't receive a MouseMove event if the mouse leaves the pane very quickly. This might lead to a "frozen tooltip" situation when the pane is destroyed but a tooltip is left behind. This call allows each pane to cleanup before it's destroyed.